### ISSUE TITLE
Multiple IntervalElement center justify with label

### ISSUE THREAD

**Issue by haorendashu** [entronad/graphic#55]
Vesion 0.9.1

Chart Code:
```
Chart(
  data: data,
  variables: {
    "type": Variable(
      accessor: (Map map) => map['type'] as String,
    ),
    "label": Variable(
      accessor: (Map map) => map['label'] as String,
    ),
    "data": Variable(
      accessor: (Map map) => map['data'] as num,
    ),
  },
  elements: [
    IntervalElement(
      position: Varset("label") * Varset('data') / Varset('type'),
      elevation: ElevationAttr(value: 0, updaters: {
        'tap': {true: (_) => 5}
      }),
      color: ColorAttr(
          variable: "type",
          values: chartInfo.getColors(),
          updaters: {
            'tap': {false: (color) => color.withAlpha(100)}
          }),
      modifiers: [DodgeModifier(ratio: 0.2, symmetric: true)],
    )
  ],
  axes: [
    Defaults.horizontalAxis,
    Defaults.verticalAxis,
  ],
  selections: {
    'tap': PointSelection(
      variable: 'label',
    )
  },
  tooltip: TooltipGuide(multiTuples: true),
  crosshair: CrosshairGuide(),
)
```

data:
```
[{"type":"title1","label":"label1","data":10.0},{"type":"title1","label":"label2","data":20.0},{"type":"title1","label":"label3","data":30.0},{"type":"title1","label":"label4","data":40.0},{"type":"title2","label":"label1","data":36.0},{"type":"title2","label":"label2","data":14.0},{"type":"title2","label":"label3","data":88.0},{"type":"title2","label":"label4","data":31.0},{"type":"title3","label":"label1","data":83.0},{"type":"title3","label":"label2","data":4.0},{"type":"title3","label":"label3","data":56.0},{"type":"title3","label":"label4","data":46.0}]
```

result:

![123](https://user-images.githubusercontent.com/1788875/155459068-107e4949-6ad9-4bc6-81ee-cc4ca68bdf93.png)

expect:

![expect](https://user-images.githubusercontent.com/1788875/155459100-13df0c25-47aa-4f8a-a525-3dd43a0f8f75.png)

how to config multiple IntervalElement center justify with label?

**Comment by entronad** [comment]
Set:
```
axes: [
    Defaults.horizontalAxis..label = LabelStyle(...),
    Defaults.verticalAxis,
  ],
```
In the LabelStyle specification, you can set any style you need for the tick label. I guess you will need to adjust `offset`, `align`, or `textAlign`. See details in:
https://pub.dev/documentation/graphic/latest/graphic/LabelStyle-class.html

**Comment by haorendashu** [comment]
Great! Thanks!

**Comment by entronad** [comment]
There is a Bug that the label won't center when dodged by default. It is fixed in v0.9.4.

You may needed to update some offset when upgrading the lib.
